src/librustdoc/html/render/mod.rs

```diff
@@ -1467,7 +1476,7 @@ fn render_impl(
                     "<div id=\"{}\" class=\"{}{} has-srclink\">",
                     id, item_type, in_trait_class
                 );
-                render_rightside(w, cx, item, containing_item);
+                render_rightside(w, cx, item, containing_item, RenderMode::Normal);
```

Comment:
Why doesn't this use `render_mode`?

Reply:
Because an associated const will always be const (unless I missed something?).

Reply:
Hmm, but is an associated const ever rendered for "Methods from Deref"? I wouldn't think so, since a const isn't a method. I'd prefer this code to always pass the actual render mode so we don't get confused later.

Reply:
Makes sense, doing it!